src/appengine/handlers/cron/schedule_corpus_pruning.py

```diff
@@ -41,24 +41,37 @@ def _get_latest_job_revision(job):
   return revisions[-1]
 
 
+def get_tasks_to_schedule():
+  """Return a list of (task_target, job_name, queue_name) tuples."""
+  tasks_to_schedule = []
+  for job in data_types.Job.query():
+    if not utils.string_is_true(job.get_environment().get('CORPUS_PRUNE')):
+      continue
+
+    if utils.string_is_true(job.get_environment().get('CUSTOM_BINARY')):
+      # Custom binary jobs do not have revisions.
+      latest_revision = None
+    else:
+      latest_revision = _get_latest_job_revision(job)
+      if not latest_revision:
+        continue
+
+    queue_name = tasks.queue_for_job(job.name)
```

Comment:
Why change the variable name from queue to queue_name?

Reply:
Found myself confused by `queue`, what type that variable was. Don't have strong opinion here though. Also unsure about `tasks_to_schedule`, maybe `tasks_arguments` or `parameters` or just `result` :)